### ISSUE TITLE
Ads before strating videos

### ISSUE THREAD

**Issue by Ghanda12** [uBlockOrigin/uAssets#5794]
on following sites (click first video on page)

```
https://www.wetter.de/cms/regen-und-wolkenfilm-fuer-zwei-tage-09-06-2019-tagsueber-meist-freundlich-4353038.html

https://www.giga.de/artikel/dark-mode-im-smartphone-nur-spielerei-mir-doch-egal/

https://www.wetter.com/videos/kolumnen/16-tage-trend-hitze-die-ausnahme/5cfa6063a5b4b92a276e1449

https://www.welt.de/politik/deutschland/article194985637/SPD-Vize-Oppermann-plaediert-fuer-Migrationspolitik-mit-knallharten-Regeln.html
```

**Comment by JobcenterTycoon** [comment]
for weitter.de `||damoh.wetter.de^$media,redirect=noopmp4-1s,domain=wetter.de`

**Comment by okiehsch** [comment]
`https://www.giga.de/artikel/dark-mode-im-smartphone-nur-spielerei-mir-doch-egal`
The pre-roll ads are blocked by the existing filter in `uBO-filters` on my end.

````
https://www.wetter.com/videos/kolumnen/16-tage-trend-hitze-die-ausnahme/5cfa6063a5b4b92a276e1449
````
The pre-roll ads are blocked by the existing filter in `uBO-filters` on my end.

````
https://www.welt.de/politik/deutschland/article194985637/SPD-Vize-Oppermann-plaediert-fuer-Migrationspolitik-mit-knallharten-Regeln.html
````
Duplicate of 
https://github.com/uBlockOrigin/uAssets/issues/5726

**Comment by JobcenterTycoon** [comment]
but the ads on `wetter.de` are not blocked

**Comment by okiehsch** [comment]
Now they are.

**Comment by okiehsch** [comment]
I take it you also can't reproduce at `wetter.com` and `giga.de`?

**Comment by JobcenterTycoon** [comment]
yes

**Comment by JobcenterTycoon** [comment]
Ads are back on `https://www.wetter.de/cms/klimawandel-sorge-um-golfstrom-und-amoc-meeresstroemungen-koennten-vor-kipppunkt-stehen-4809478.html`
